### PR TITLE
Updated PYTHON to PYTHON_VER

### PR DESCRIPTION
**Pull request by jolorunyomi**

### CODE REVIEW

File: conda/recipes/cucim/build.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ cp -P python/install/lib/* python/cucim/src/cucim/clara/
 
 pushd python/cucim
 
-echo "PYTHON: ${PYTHON}"
-$PYTHON setup.py install
+echo "PYTHON: ${PYTHON_VER}"
+$PYTHON_VER setup.py install
```
Comment (jakirkham) [inline conda/recipes/cucim/build.sh:29]: Should this still be `$PYTHON` as it is referring to the executable and not the version?

**Review by jakirkham — COMMENTED** [review]
Thanks Joseph! 😀

Had one question below. Otherwise looks good 🙂

**Comment by jakirkham** [comment]
JFYI am retargeting to 21.12 per conversation offline

cc @ajschmidt8 (for awareness)

**Comment by ajschmidt8** [comment]
Closed since there is new pipeline work being done that will supersede these changes.

**Pull request closed without merging**